nemo_text_processing/inverse_text_normalization/es/taggers/money.py

```diff
@@ -66,22 +86,31 @@ def __init__(self, cardinal: GraphFst, decimal: GraphFst):
 
         add_leading_zero_to_double_digit = (NEMO_DIGIT + NEMO_DIGIT) | (pynutil.insert("0") + NEMO_DIGIT)
 
+        one_graph = pynini.union("un", "una").optimize()
```

Comment:
Memory is blanking, but I recall that some currencies use `uno` in place of `un`. Could you confirm or deny?

Reply:
can confirm this case should not be seen by the grammar (unless ASR is having significant issues)